### DISCUSSION TITLE
Seperation of users and organisation in search results

### DISCUSSION THREAD

**Discussion by Deadlyelder** [community/community#31988] [Other Feature Feedback, Questions, & Ideas]
Within the UI, when searching for a keyword, we can filter it with user search but not specifically with organisation filter. For instance, searching for GitHub shows user + organisations despite selecting users within the filter column. Ideally, we should be able to separate the organisation and users from the results.
We can use keywords with the search (such as the `org:`) to filter organisations, it would be good to have it directly within the search interface, or better yet, have a way to distinguish results from the users vs. organisations.

**Comment by Akash1134** [comment]
Hi @Deadlyelder Thanks for the feedback. I have added the relevant tag to the discussion so that our product team can review it.
